scripts/deploy_terraform.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cd terraform/ && pwd
+
+# Terraform workflow
+terraform init --reconfigure
+terraform plan -var-file tfvars/main.terraform.tfvars
+terraform apply -var-file tfvars/main.terraform.tfvars
```

Comment:
Do we need --auto-approve here?

Reply:
Could add it, but as it's running locally and not in CI, I didn't feel it was necessary. Would let us review the plan before we apply